packages/core/src/clocks.ts

```diff
@@ -0,0 +1,46 @@
+﻿import * as R from 'ramda'
+import { Clock } from './types'
+import A from 'automerge'
+import { Map } from 'immutable'
+/**
+ * A vector clock is a map, where the keys are the actorIds of all actors that have been active on a
+ * particular document, and the values are the most recent sequence number for that actor. The
+ * sequence number starts at 1 and increments every time an actor makes a change.
+ */
+
```

Comment:
this file just consolidates clock-related stuff that's needed in more than one place.